llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1579,6 +1587,11 @@ bool AddressSanitizer::GlobalIsLinkerInitialized(GlobalVariable *G) {
 void AddressSanitizer::instrumentPointerComparisonOrSubtraction(
     Instruction *I, RuntimeCallInserter &RTCI) {
   IRBuilder<> IRB(I);
+
+  if(CLAsanDormant)
+    IRB.SetInsertPoint( SplitBlockAndInsertIfThen(IRB.CreateNot(IRB.CreateLoad(IRB.getInt1Ty(), DormantAsanFlag)), I, false));
```

Comment:
Does Int1Ty mean bool ?

Reply:
Yeah this is how LLVM IR handles bools. One bit int.